app/README.md

```diff
@@ -9,4 +9,12 @@ $ pipenv install
 ```
 $ pipenv shell
 (app) python app.py
-```
\ No newline at end of file
+```
+
+## Create DataBase
+- If you need to setup your database you will need to do the following inside a python repl while in your pipenv shell.
+- Make sure to be within the `app` folder directory
+```
+>>> from fileName import db
+>>> db.create_all()
+```
```

Comment:
Add reminder that a new `app.sqlite` file will be created afterwards.

Reply:
ok ill work on that